src/auth/screens/login.screen.js

```diff
@@ -223,8 +223,8 @@ class Login extends Component {
     return (
       <View style={styles.browserLoader}>
         <Text style={styles.browserLoadingLabel}>
-          {this.state.loaderText}
-        </Text>
+          {' '}{this.state.loaderText}{' '}
+        </Text>{' '}
```

Comment:
@Ernoff @housseindjirdeh Why are there so many {' '} additions in this file?

Reply:
Must be keyboard glitch. Apologies

Reply:
#410 

This cleans it up